stratum/hal/lib/common/yang_parse_tree_paths.cc

```diff
@@ -357,6 +419,47 @@ U GetValue(
   return resp;
 }
 
+// Optical-port-specific version.
+template <typename T, typename U>
+U GetOpticalChannelValue(
```

Comment:
Why is this not called `GetValue` like the others?

Reply:
I didn't noticed that we have this function. I only refactored the `SetOpticalChannelValue` one. Will also refactor this one.

Reply:
Actually we do have another special function for the counter only. But we should simplify that also.

Reply:
Should be fine to keep these functions now and refactor entire file in another patch